packages/cerebral/devtools.d.ts

```diff
@@ -0,0 +1,13 @@
+export interface DevOptions {
```

Comment:
Do we need to export this interface? Take a note that all exported interfaces becomes part of public API and it will be covered by semver. Eg,, renaming exported interface or any other breaking chage in it would require major bump.